Sources/ArgumentParser/Usage/HelpGenerator.swift

```diff
@@ -353,8 +353,10 @@ func _terminalSize() -> (width: Int, height: Int) {
   return (80, 25)
 #elseif os(Windows)
   var csbi: CONSOLE_SCREEN_BUFFER_INFO = CONSOLE_SCREEN_BUFFER_INFO()
-
-  GetConsoleScreenBufferInfo(GetStdHandle(STD_OUTPUT_HANDLE), &csbi)
+  // If the function fails, the return value is zero.
+  // https://docs.microsoft.com/en-us/windows/console/getconsolescreenbufferinfo
+  let err = GetConsoleScreenBufferInfo(GetStdHandle(STD_OUTPUT_HANDLE), &csbi)
+  guard err != 0 else { return (80, 25) }
```

Comment:
I would just fold this into a single statement:

```suggestion
guard GetConsoleScreenBufferInfo(GetStdHandle(STD_OUTPUT_HANDLE), &csbi) else {
  return (80, 25)
}
```